### DISCUSSION TITLE
Pixel 6a Android 12 possible alert issues

### DISCUSSION THREAD

**Discussion by jdewolfe** [NightscoutFoundation/xDrip#2957] [Q&A]
I'm concerned as lately I've been sleeping through all of my low blood sugar alerts.

I have continuous alerts that are quite loud and work during the day.  One is set at 70 mg/dl and another is set at 67 mg/dl with a different type of alarm.

The general pattern in logs is Alert, SnoozeOnNotificationDismissService, Snoozed Alert for 35 minutes.

However, this seems to be happening during my sleep.

Generally before bed I will run a test alert to make sure that it's working and silence it by swiping the message box to the right.

Default snooze is 35 minutes.

I wish there was a nice way to know whether the phone is being noisy other than recording through the night.

1:08 Alert, 1:09 Snooze until 1:44, 1:44 Alert, 1:44 snooze until 2:19, 2:19 Alert, 2:19 Snooze until 2:54, 2:54 Alert, 2:55 Snooze until 3:30, whole ton of Nightscout treatments, number went up beyond alert levels, 4:38 Alert, 4:39 snoozed until 5:14, 5:14 Alert, snoozed until 5:49...for each of these the SnoozeOnNotificationDissmissService called source alerts shown for time is a little different.

I woke up at 7 AM perfectly rested although the number was fairly low for half the night meaning I don't think it was possible to wake up every 35 minutes for the snoozings.

I also wear a Bluejay watch in case that affects anything.

version 344deb4-2022.11.16 although going to update in case it helps

**Comment by Navid200** [comment]
I take it there is no one who sleeps nearby.

Can you set up a sound recorder tonight?  If the logs show an alarm went off at some particular time but you don't remember it, let's listen to the recording for exactly that time.

BlueJay can silence an alert if you press on the button.

If the logs show an alert went off and was snoozed, it means someone snoozed it.  But, you don't remember.  Is it possible that you just don't remember?  Again, a sound recorder would answer that.

Do you have Nightscout?
If you do, you can leave a computer on with Nightscout showing in a browser.  If you leave the Nightscout alerts at default in the browser, the computer will alert you for safety so that you won't miss the alerts.  Although, the exact value of the alert may not match yours in xDrip.  So, you can adjust that.

Can you post a screenshot of the logs showing the alert?

Can you also post a screenshot of the xDrip alert settings page so that we can see everything like overwrite silent mode etc.?

**Comment by Navid200** [comment]
Please also post a screenshot of the following xDrip page.

Settings &#8722;> Alarms and Alerts &#8722;> Glucose Alerts Settings

**Reply by jdewolfe** [reply]
Thank you for response.

Last night after upgrading I was awoken immediately by the first alarm.  No other changes were made.

I do recall a couple prior evenings of not being woken up by alarms but figured that I had slept through them whereas this one being all night I decided to investigate further.

I wonder if the watch affected things.
![0](https://github.com/NightscoutFoundation/xDrip/assets/28075111/e73ea72d-a03b-4d7f-bb61-4b7fadc98d6f)
![1](https://github.com/NightscoutFoundation/xDrip/assets/28075111/ce235e67-441d-4ac2-80e5-3f5be488e9f2)
![2](https://github.com/NightscoutFoundation/xDrip/assets/28075111/d34d8651-e987-4c10-b93c-32ea673c7d85)
![3](https://github.com/NightscoutFoundation/xDrip/assets/28075111/d4f7de6f-dc0d-42ae-8dce-8e2b519fa3a7)
![4](https://github.com/NightscoutFoundation/xDrip/assets/28075111/8103e0c8-570a-4781-ac24-e6f3d7ed9fdd)


Photos attached.

**Comment by Navid200** [comment]
I always shut down my BlueJay when I sleep.  But, I have never heard anyone claiming that BlueJay would snooze an alert on its own.
